loadtest/control/simplecontroller/controller.go

```diff
@@ -60,7 +62,14 @@ func (c *SimpleController) Run() {
 
 	c.status <- control.UserStatus{ControllerId: c.id, User: c.user, Info: "user started", Code: control.USER_STATUS_STARTED}
 
-	defer c.sendStopStatus()
+	defer func() {
```

Comment:
My head is burning at the moment but isn't the function `(c *SimpleController) Run()`  suppose to run asynchronously? We should be waiting like [this](https://github.com/mattermost/mattermost-load-test-ng/blob/0a05712486e69ffa2dfb1c366bd33bb6bc884559/cmd/loadtest/loadtest.go#L81). If so, this function should be re-considered.

Reply:
That's `loadtest.Run()`. 
`controller.Run()` blocks and it should be used inside a goroutine.
It's called here:

https://github.com/mattermost/mattermost-load-test-ng/blob/0a05712486e69ffa2dfb1c366bd33bb6bc884559/loadtest/loadtest.go#L82-L85

Reply:
You just confirmed that my head is burning, thanks 👍